pkg/apis/config/v1alpha1/types.go

```diff
@@ -146,6 +146,8 @@ type EdgeProxyConfig struct {
 	Socks5Proxy *Socks5Proxy `json:"socks5Proxy,omitempty"`
 	// LoadBalancer indicates the load balance strategy
 	LoadBalancer *LoadBalancer `json:"loadBalancer,omitempty"`
+	// InvertServiceFilter indicates if the service filtering is inverted
+	InvertServiceFilter bool `json:"invertServiceFilter,omitempty"`
```

Comment:
We need to think a bit more about the variable name and try to be as close to its intent as possible.

Would ServiceFilterMode be better? For example:
```
ServiceFilterMode string  // option ["LabelExistsThenFilter", "LabelDoesNotExistsThenFilter"]
```

Reply:
Sounds good to me. I'd say we make `FilterIfLabelExists` is the default value if `ServiceFilterMode` is unset.

Reply:
Sounds good!